hardhat.tasks.js

```diff
@@ -20,37 +18,20 @@ task("tenderly-verify", "verifies current deployment on tenderly").setAction(
   },
 );
 
-task("set-ship-state", "sets restrictions to closed, open, onlyShip or onlySafeWithShip")
-  .addPositionalParam("state")
-  .setAction(async (args, { getNamedAccounts, ethers }) => {
-    const { deployer } = await getNamedAccounts();
-    const podEnsRegistrar = await ethers.getContract("PodEnsRegistrar", deployer);
-
-    const stateEnum = {
-      onlySafeWithShip: 0,
-      onlyShip: 1,
-      open: 2,
-      closed: 3,
-    };
-
-    const { state } = args;
-
-    const currentState = await podEnsRegistrar.state();
-    if (currentState === stateEnum[state]) {
-      console.log("Contract was already set to that state");
-      return;
-    }
-    await podEnsRegistrar.setRestrictionState(stateEnum[state]);
-    console.log(`Successfully changed state to ${state}`);
-  });
```

Comment:
❓ Did we decide we no longer need to maintain this state?

Reply:
yah, we are no longer using the $SHIP token gating, the only thing we're gating is frontend via the permit NFTs